docs/howto.md

```diff
@@ -188,6 +188,94 @@ future. Do note that by separating the credentials in the secret, the password
 is never directly accessed by the operator itself.
 
 
+# Use a cluster internal share as persistent volume (without auto-provisioning)
+
+Kubernetes does not by default support mounting `smb` shares.  For now
+it's neccessary to install the [smb csi
+driver](https://github.com/kubernetes-csi/csi-driver-smb).  Please refer
+to its documentation on how to install the csi driver.
+
+First create a (non provisioning) storage class to differentiate smb shares
+from other storage.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: smb
+parameters:
+  type: smb
+provisioner: kubernetes.io/no-provisioner
+reclaimPolicy: Retain
+volumeBindingMode: Immediate
+```
+
+Then create a secret containing the username and password to mount the share.
+These are AD credentials when using AD authentication or the username and
+passwords as configured above.
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: myshare-mount-creds
+  namespace mynamespace
+type: Opaque
+stringData:
+  username: user1
+  password: T0Psecre7
+```
+
+The following persistent volume will allow mounting the share.
+```
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-mynamespace-myshare
+spec:
+  capacity:
+    storage: 1Gi
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  mountOptions:
+    - dir_mode=0777
+    - file_mode=0777
+    - vers=3.0
+  csi:
+    driver: smb.csi.k8s.io
+    readOnly: false
+    volumeHandle: mynamespace-myshare  # make sure it's a unique id in the cluster
+    volumeAttributes:
+      source: "//myshare.mynamespace/My Great Share"
```

Comment:
I think we need to elaborate a little bit more on how to construct this string. Can you add a short paragraph explaining where a user who is not super-familiar with samba or our operator can get the values for "myshare.mynamespace" and "My Great Share"?  Everything else is largely "copy-n-paste" but this line is particularly critical to have matching so I think it would be good to explain a bit more in detail.